Task_HelloWorld/index.html

```diff
@@ -1,15 +1,16 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Hello World Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
```

Comment:
The 'top' attribute should be specified using CSS, e.g., 'style="top: 0;"' instead of 'top="0"'